olmo/version.py

```diff
@@ -1,5 +1,5 @@
 _MAJOR = "0"
-_MINOR = "3"
+_MINOR = "4"
 # On main and in a nightly release the patch should be one ahead of the last
 # released build.
 _PATCH = "1"
```

Comment:
```suggestion
_PATCH = "0"
```

Reply:
I thought we kept the repo at version + 1 (so v0.4.1 when we are at 0.4.0)

Reply:
No we typically leave it as is until the next release